go/vt/servenv/pprof.go

```diff
@@ -299,32 +299,45 @@ func (prof *profile) init() (start func(), stop func()) {
 	}
 }
 
```

Comment:
This diff isn't very readable.  It may be easier to just look at the file starting at line 300.